marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/flow/createFlow.sjs

```diff
@@ -16,10 +16,20 @@
 'use strict';
 
 const Artifacts = require('/data-hub/5/artifacts/core.sjs');
+const ds = require("/data-hub/5/data-services/ds-utils.sjs");
 
 var name;
 var description;
 
+let existingFlow = null;
+try {
+  existingFlow = Artifacts.getArtifact('flow', name);
+} catch (e) {
+  // swallowing error, as we don't want there to be an existing flow with the given name.
+}
+if (existingFlow) {
+  ds.throwBadRequest(`Cannot create flow; flow with the name '${name}' already exists`);
+}
```

Comment:
Can you add a small test to manageFlow.sjs as well? Generally, an HC test alone won't suffice - we want to have coverage of ML code within the DHF core project.